charts/model-engine/templates/service_config_map.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
 data:
   launch_service_config: |-
     dd_trace_enabled: {{ .Values.dd_trace_enabled | default false | quote }}
+    gateway_namespace: {{ .Release.Namespace }}
```

Comment:
do we need to pass it through that ` | quote` thing? we could helm install --dryrun to see what the service config map ends up as

Reply:
(or `just _deploy training true` actually heh which is probably easier)